.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+        flutter doctor -v
+        
+    - name: Create test results directory
+      run: mkdir -p test_results/performance
+      
+    - name: Run performance tests
+      run: |
+        dart test/performance/ci_runner.dart
+      continue-on-error: true
+      
+    - name: Upload performance reports
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)